commit ae354d80b25d90576f37222b6500f0e7ac45405b
Author: Remco Burema <r.burema@ultimaker.com>
Date:   Sun Sep 6 20:25:51 2020 +0200

    Add missing header to sparse-grid test file.
    
    Fixes #1318

diff --git a/tests/utils/SparseGridTest.cpp b/tests/utils/SparseGridTest.cpp
index f62d2706..ca263578 100644
--- a/tests/utils/SparseGridTest.cpp
+++ b/tests/utils/SparseGridTest.cpp
@@ -2,6 +2,7 @@
 //CuraEngine is released under the terms of the AGPLv3 or higher.
 
 #include <gtest/gtest.h>
+#include <algorithm>
 #include <unordered_set>
 #include <vector>
 

