--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -141,6 +141,8 @@
 add_executable(CuraEngine src/main.cpp) #Then compile main.cpp as separate executable, and link the library to it.
 target_link_libraries(CuraEngine _CuraEngine)
 
+set_target_properties(CuraEngine PROPERTIES COMPILE_DEFINITIONS "VERSION=\"${CURA_ENGINE_VERSION}\"")
+
 # Compiling the test environment.
 if (BUILD_TESTS)
     message(STATUS "Building tests...")
